features/steps/steps.py

```diff
@@ -98,6 +149,19 @@ class representation_type_error:
     def __str__(self):
         return f"On instance {fmt(self.inst)} the {self.representation_id} shape representation does not have {self.representation_type} as RepresentationType"
 
+
+@dataclass
+class polyobject_structure_error:
```

Comment:
I'm a bit in doubt on whether we shouldn't simply use 2 different dataclasses. Similar comment I made to @Ghesselink earlier. I guess they should be as simple as possible and maybe later we assign an error code to them. But I don't have a very strong opinion on this.

Reply:
I don't have a strong opinion either. Separated them here.